v2/cli/init_command.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type initFields struct {
+	ProjectID   string
+	Language    string
+	GitCloneURL string
+	Script      string
+}
```

Comment:
I usually recommend against creating types that are only used in one place because it "clutters" things.

Also, not every template that you render needs all of this information.

I would suggest using an anonymous struct type having only relevant fields anywhere you need to pass stuff to `execTemplate`.

Reply:
Sounds good. If so, would it be good to just save each value to a local variable and pass them into each execTemplate as needed?

Reply:
For now, I just have fields saved as an anonymous struct, and pass them into execTemplate in the same way. Although there are unnecessary fields for some templates, I feel like it might be too cluttered to declare an entire different anonymous struct for each execTemplate. I could be wrong in my thinking though

Reply:
> For now, I just have fields saved as an anonymous struct, and pass them into execTemplate in the same way

> I feel like it might be too cluttered to declare an entire different anonymous struct for each execTemplate

This sounds like a reasonable compromise.